tests/test_sessions.py

```diff
@@ -662,25 +662,14 @@ class SessionNoSlots(nox.sessions.Session):
 
         session = SessionNoSlots(runner=runner)
 
-        with mock.patch.object(session, "_run", autospec=True) as run:
-            with pytest.warns(
-                FutureWarning,
-                match=(
-                    r"use of session\.install\(\) is deprecated since it would modify"
-                    r" the global Python environment"
-                ),
-            ):
-                session.install("requests", "urllib3")
-            run.assert_called_once_with(
-                "python",
-                "-m",
-                "pip",
-                "install",
-                "requests",
-                "urllib3",
-                silent=True,
-                external="error",
-            )
```

Comment:
I removed the `assert_called_once_with` because `session._run` is no longer used when blocking the installation, am I right?

Reply:
Makes sense.